Doppler.Sap/Mappers/Billing/BillingForUsMapper.cs

```diff
@@ -65,6 +65,9 @@ public bool CanMapSapSystem(string sapSystem)
 
         public SapSaleOrderModel MapDopplerBillingRequestToSapSaleOrder(BillingRequest billingRequest)
         {
+            var date = billingRequest.InvoiceDate ?? _dateTimeProvider.GetDateByTimezoneId(_dateTimeProvider.UtcNow, _timezoneConfig.InvoicesTimeZone);
```

Comment:
what do you think add in this line the command .ToString("yyyy-MM-dd") ?

Reply:
Done